packages/panels/resources/views/components/sidebar/index.blade.php

```diff
@@ -34,7 +34,7 @@
         @endif
     @endif
     @class([
-        'fi-sidebar fixed inset-y-0 start-0 z-30 grid h-screen content-start bg-white transition-all dark:bg-gray-900 lg:z-0 lg:bg-transparent lg:shadow-none lg:ring-0 lg:transition-none dark:lg:bg-transparent',
+        'fi-sidebar fixed inset-y-0 start-0 z-30 grid h-screen content-start bg-white transition-all dark:bg-gray-900 lg:bg-transparent lg:shadow-none lg:ring-0 lg:transition-none dark:lg:bg-transparent',
```

Comment:
Why aren't we changing `z-30` here to `z-40` instead?
Haven't tested, but I'm just asking to prevent breaking any edge cases.

Also, have you tested this change with modals?
Modals also have `z-40`, so make sure this PR doesn't break anything there. 🙂

Since I don't have a simple reproduction repository, I can't actually test the change.

Reply:
@zepfietje you are right it didnt need to change. It can stay as ``z-30``. What made the difference was removing ``lg:z-0``. Just made a change with both ``z-30`` and ``z-40`` and the modals look okay.

Here are some screenshots

![Screenshot 2023-12-28 at 8 15 10 am](https://github.com/filamentphp/filament/assets/22025439/b7c8ca32-c586-4640-b5d4-747f01729d8c)
![Screenshot 2023-12-28 at 7 48 26 am](https://github.com/filamentphp/filament/assets/22025439/9c2df445-f517-4986-995e-73a71eb2d1d5)
![Screenshot 2023-12-28 at 8 18 10 am](https://github.com/filamentphp/filament/assets/22025439/f381ae7e-2048-415a-9476-005a43123027)

I reverted back to ``z-30`` to have less changes in this PR
Will upload the reproduction repo in just a moment

Reply:
@zepfietje Here is the repo

username: admin@admin.com
password: password

https://github.com/danieladarve/filament-overflow-issue

Reply:
Thanks, @danieladarve!